code/modules/sanity/inspiration_component.dm

```diff
@@ -72,6 +72,12 @@
 			if("Cognition")
 				stat_noun = pick("cognition", "intelligence", "knowledge")
 				stat_adjective = pick("smart", "intelligent", "concentrated", "knowledgable")
+			if("Vivification")
+				stat_noun = pick("invigoration", "vigorous", "healthy", "fit")
+				stat_adjective = pick("vitality", "vigour")
+			if("Anatomy")
+				stat_noun = pick("physique", "soma", "build", "frame")
```

Comment:
none of these nouns are particularly suggestive of like. "can handle a lot of drugs; knows a lot about anatomy". probably because that's already been taken by bio.